requirements.txt

```diff
@@ -1,76 +1,88 @@
 #
-# This file is autogenerated by pip-compile
```

Comment:
@yarongilor I don't think all these edits should be in this file.

Reply:
file is rolled back to original state.